### PR TITLE
Internal updates for TT

### LINKED ISSUE #214: Remove search for internal build
search can't index the internal pages and will just link to the external site. Need to remove from sidebar for internal build

### PR DESCRIPTION
**Pull request by alisonjoseph**
Ref #216
closes #214
- Increases sidebar width from 200px to 240px for external/internal sites
- Updates "About Carbon" nav item to "About"

**Updates for the internal site only**
- new header banner
- Site title says "Product Design" instead of "Carbon Design System"

**Testing**
Should test internal and external versions, only changes that should appear on external site is the increase in the navigation width & the nav link for About.
`yarn dev`
`yan dev:internal`

### CODE REVIEW

File: src/components/SideNav/side-nav.scss [hunk 1]
```diff
@@ -5,11 +5,11 @@
   @include typescale('zeta');
   @include layer('temporary-nav');
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-direction: column; 
   background-color: $ui-01;
   border-right: 2px solid $ui-02;
   width: 12.5rem;
+  width: rem(240px);
```
Comment (asudoh) [inline src/components/SideNav/side-nav.scss]: Did you mean to remove the preceding line?

File: src/styles/_home.scss [hunk 2]
```diff
@@ -400,3 +400,22 @@
   left: 4rem;
   z-index: 1;
 }
+
+
+// Internal only
+.overview-page__banner.overview-page__banner--internal {
+  background: #171717;
```
Comment (asudoh) [inline src/styles/_home.scss]: `$ibm-colors__gray--100`?

File: src/styles/_home.scss [hunk 3]
```diff
@@ -400,3 +400,22 @@
...
+  color: #F3F3F3;
```
Comment (asudoh) [inline src/styles/_home.scss]: `$ibm-colors__gray--10`?

File: src/pages/index.js [hunk 4]
```diff
@@ -16,6 +16,318 @@ import scatter from '../content/homepage/images/scatter.svg';
...
+    aria-label="overview page banner">
+    <h1 className="banner__logo">
+      Product Design
+    </h1>
+  </section>
+) : (
+  <section
+    className="overview-page__banner"
+    aria-label="overview page banner">
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2500 823">
```
Comment (asudoh) [inline src/pages/index.js]: Future thought - Making this `<svg>` a React component?

**Comment by alisonjoseph** [comment]
Thanks for the comments @asudoh, will clean up the code and make adjustments today. 

Would definitely be good to "componentize" the homepage, however since the designers are working on a redesign for our next release might make sense to wait and see what they come up with. Would also like to adjust the homepage to pull from a markdown file at some point.

**Comment by alisonjoseph** [comment]
@designertyler new preview link http://exotic-sense.surge.sh/design/product/

Updated the background color, also made sure the buttons on the left were the same color blue, they used to be but somehow that got missed in the re-build. 

Looked at adding a line break in the nav, and can't really do that easily the way that its generated. Could add some padding on the right, but then it doesn't break in the middle and only the last word wraps. Not sure how much time I should spend trying to get that working.

**Comment by alisonjoseph** [comment]
@designertyler fixed

**Pull request merged**